addons/pos_self_order/static/src/app/components/kiosk_attribute_selection/attribute_selection.xml

```diff
@@ -15,7 +15,7 @@
                                 t-attf-class="{{ valueSelected ? 'o_kiosk_box_selected text-bg-primary': '' }}">
                                     <span t-esc="value.name" class="fs-4"/>
                                     <span t-if="shouldShowPriceExtra(value)" t-esc="formatExtraPrice(value)" class="fs-5 text-muted"
-                                    t-att-class="{ 'text-light opacity-75' : valueSelected }"  />
+                                    t-att-class="{ 'text-reset opacity-75' : valueSelected }"  />
```

Comment:
I noticed we don't use the same active effect on a variant than we use on a combo, I think it would be more consistent to style them the same way as combo. Making the border active, using a badge for the price and contrasting the text color.
![image](https://github.com/user-attachments/assets/82d61541-7bb6-470c-9af8-3f3b8557adf0)